packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -131,15 +131,23 @@ export const TextInput = withTheme(
       [onChange]
     );
 
+    const hasError = typeof error !== "undefined";
     const ariaDescribedBy = useCx(
       hint && `${oid}-hint`,
-      typeof error !== "undefined" && `${oid}-error`
+      hasError && `${oid}-error`
     );
 
+    const ariaProps =
```

Comment:
Apologies for the strange formatting here -- this seems to be automatically applied by tooling rather than by my editor/IDE.